easybuild/easyconfigs/p/pgi/pgi-15.7-GNU-4.9.2-2.25.eb

```diff
@@ -0,0 +1,22 @@
+name = 'pgi'
```

Comment:
@bartoldeman: we try to stick to the 'official' software name; I think `PGI` is a lot more common than `pgi`, so can we change this (along with the easyconfig filename)?